blog/_posts/2023-03-20-curation-model.md

```diff
@@ -0,0 +1,72 @@
+
+<div class="text-center">
+    <img class="img-thumbnail" src="{{ site.baseurl }}/blog/images/ads-curation_model_f2.png" />
+<em>Figure 2: Directed network based on citation relationship between journals. The nodes represent journals. This is a fictitious network, for illustration purposes only.
+</em>
+</div>
+<br>
+
+At its core, the algorithm that calculates the Eigenfactor values is the well-known [PageRank](https://en.wikipedia.org/wiki/PageRank) algorithm, which properly deals with the fact that journals are all interconnected in a citation network (see figure 2, showing a fictitious network). The [Journal Citation Reports](https://en.wikipedia.org/wiki/Journal_Citation_Reports) (JCR), published by Clarivate Analytics, provides the Journal Eigenfactor on an annual basis. However, we prefer not to have to depend on an external data source for our decisions. That is why we made a comparison between Journal Eigenfactors calculated using ADS citation data and those listed in the JCR for a number of years.
+
+The results are shown in figure 3. The fact that values are not equal is no surprise, because the citation networks in the ADS and Web of Science are different. The encouraging result is that the values have a very strong correlation.
+
+<div class="text-center">
+    <img class="img-thumbnail" src="{{ site.baseurl }}/blog/images/ads-curation_model_f3.png" />
+<em>Figure 3: Comparison between Eigenfactors calculated from ADS data and provided in the JCR. The categorization is based on journal assignments provided in the JCR.
+</em>
+</div>
+<br>
+
+Next, we compare journal ranking using Eigenfactor values. In other words, if we create a ranked list of journals, how does, say, the top 100 compare with one based on JCR values? In the end, this is what we are looking for. The actual Eigenfactor values are not relevant, since we are only interested in relative values. It is the ranked list that will provide us with the candidates for the Core Collection. We generated and compared the top 100 for the period 2010 thru 2021. For the first decade the top 100 lists are identical; for 2020 and 2021 they only differ in the very tail. For 2020 the ADS list contains the [Bulgarian Astronomical Journal](https://www.astro.bas.bg/AIJ/) (bibstem BlgAJ), where the JCR has [Astrophysics](https://www.springer.com/journal/10511) (bibstem Ap). For 2021 the ADS list contains [Contributions of the Astronomical Observatory Skalnate Pleso](https://www.ta3.sk/caosp/) (bibstem CoSka), where the JCR has again [Astrophysics](https://www.springer.com/journal/10511) (bibstem Ap). For both journals, present in the ADS list, WoS is highly incomplete.
```

Comment:
* Is this just astro, or astro+physics again? If just astro, I'd definitely remove the mention of physics from way up above.
* The WoS mention is out of left field - if you want to do the comparison, I'd add a sentence explaining that a bit more